includes/html/pages/apps/overview.inc.php

```diff
@@ -11,7 +11,7 @@
 $graph_array_zoom['width']  = '400';
 $graph_array['legend']      = 'no';
 
-foreach (Application::query()->hasAccess(Auth::user())->with('device')->orderBy('app_type')->get()->groupBy('app_type') as $type => $groupedApps) {
+foreach (Application::query()->hasAccess(Auth::user())->with('device')->orderBy('app_type')->get()->sortBy('show_name', SORT_NATURAL|SORT_FLAG_CASE)->groupBy('app_type') as $type => $groupedApps) {
```

Comment:
Switch to ObjectCache while at it?

Reply:
it's in a site, not in layout menu, also completly different from application_list in objectcache.